OpenRobertaServer/staticResources/js/main.js

```diff
@@ -92,6 +93,9 @@ require.config({
         'interpreter.interpreter': 'app/nepostackmachine/interpreter.interpreter',
         'interpreter.aRobotBehaviour': 'app/nepostackmachine/interpreter.aRobotBehaviour',
         'interpreter.robotWeDoBehaviour': 'app/nepostackmachine/interpreter.robotWeDoBehaviour',
+        'interpreter.robotOrbBehaviour' : '../app/nepostackmachine/interpreter.robotOrbBehaviour',
+        
```

Comment:
Same for this, this is not used and can be removed, only the line after that is necessary